cypress/integration/invoices/invoices.createCreditInvoice.spec.js

```diff
@@ -0,0 +1,71 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import NewInvoiceLine from '../../support/fragments/invoices/newInvoiceLine';
+import Invoices from '../../support/fragments/invoices/invoices';
+import TestType from '../../support/dictionary/testTypes';
+import VendorAddress from '../../support/fragments/invoices/vendorAddress';
+import NewFund from '../../support/fragments/finance/funds/newFund';
+import Funds from '../../support/fragments/finance/funds/funds';
+import { getCurrentFiscalYearCode } from '../../support/utils/dateTools';
+import Helper from '../../support/fragments/finance/financeHelper';
+import Transaction from '../../support/fragments/finance/fabrics/newTransaction';
+
+describe('ui-invoices: Credit Invoice creation', () => {
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
+  const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
+  const fund = { ...NewFund.defaultFund };
+  const subtotalValue = 100;
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getOrganizationApi({ query: `name=${invoice.vendorName}` })
+      .then(({ body }) => {
+        invoice.accountingCode = body.organizations[0].erpCode;
+        Object.assign(vendorPrimaryAddress,
+          body.organizations[0].addresses.find(address => address.isPrimary === true));
+      });
+    cy.getBatchGroups()
+      .then(({ body }) => {
+        invoice.batchGroup = body.batchGroups[0].name;
+      });
+    Funds.createFundViaUI(fund)
+      .then(
+        () => {
+          Funds.addBudget(100);
+          Funds.checkCreatedBudget(fund.code, getCurrentFiscalYearCode());
+        }
+      );
+    invoiceLine.subTotal = -subtotalValue;
+    cy.visit(TopMenu.invoicesPath);
+  });
+
+  it('C343209 Create a credit invoice', { tags: [TestType.smoke] }, () => {
+    Invoices.createDefaultInvoiceViaUi(invoice, vendorPrimaryAddress);
+    Invoices.createInvoiceLine(invoiceLine, false);
```

Comment:
can't see "sub-total value equals -100" implementation near this step

Reply:
It's in precondition, where we're creating invoice line object

Reply:
which line? can't see verification of -100 in preconditions too

Reply:
oh, get it. I don't actually think this verification is necessary in this test, but I will add it.

Reply:
may be the best way - clear test in testrail from this check